src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -34,20 +35,38 @@
    {:id :pending :label (i18n/label :t/pending) :accessibility-label :pending-tab}
    {:id :opened :label (i18n/label :t/opened) :accessibility-label :opened-tab}])
 
+(defn empty-state-content
+  [selected-tab]
+  (case selected-tab
+    :joined
+    {:title       (i18n/label :t/no-communities)
+     :description [:<>
+                   [rn/text {:style {:text-decoration-line :line-through}}
+                    (i18n/label :t/no-communities-description-strikethrough)]
+                   " "
+                   (i18n/label :t/no-communities-description)]
+     :image       (resources/get-image (theme/theme-value :no-communities-light
+                                                          :no-communities-dark))}
+    :pending
+    {:title       (i18n/label :t/no-pending-communities)
+     :description (i18n/label :t/no-pending-communities-description)
+     :image       (resources/get-image (theme/theme-value :no-pending-communities-light
+                                                          :no-pending-communities-dark))}
+    :opened
+    {:title       (i18n/label :t/no-opened-communities)
+     :description (i18n/label :t/no-opened-communities-description)
+     :image       (resources/get-image (theme/theme-value :no-opened-communities-light
+                                                          :no-opened-communities-dark))}))
```

Comment:
`case` needs a default option, otherwise it throws an exception

Reply:
Just added `nil` since we currently have a fixed set of fixed tabs, thanks for pointing :+1: